### DISCUSSION TITLE
hello, why this project use next.js and hono

### DISCUSSION THREAD

**Discussion by maobase** [unkeyed/unkey#1153] [Q&A] [answered]
hello, why this project use next.js and hono,all base on bun runtime?

**Comment by chronark** [comment, marked as answer]
We use nextjs for the landing page and dashboard and hono for the api, because both provide really good DX and are powerful.
We don't use bun.

**Reply by maobase** [reply]
why do not use bun? bun has best performence

**Reply by chronark** [reply]
Neither vercel, nor cloudflare can run bun
